src/main/java/lottery/domain/LotteryTicket.java

```diff
@@ -0,0 +1,41 @@
+package lottery.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LotteryTicket {
+
+    public static final int LOTTERY_NUMBER_COUNTS = 6;
+    private final List<LotteryNumber> lotteryNumbers;
+
+    public LotteryTicket(List<LotteryNumber> lotteryNumbers) {
+        validateLotteryNumbers(lotteryNumbers);
+        this.lotteryNumbers = lotteryNumbers;
+    }
+
+    public List<Integer> getLotteryNumbers() {
+        return lotteryNumbers.stream()
+                .map(LotteryNumber::getLotteryNumber)
+                .collect(Collectors.toList());
+    }
+
+    private void validateLotteryNumbers(List<LotteryNumber> lotteryNumbers) {
+        if (lotteryNumbers.size() != LOTTERY_NUMBER_COUNTS) {
+            throw new IllegalArgumentException(ErrorMessages.INVALID_NUMBER_COUNTS);
+        }
+        if (isDuplicatedLotteryNumbers(lotteryNumbers)) {
+            throw new IllegalArgumentException(ErrorMessages.DUPLICATED_NUMBER);
+        }
+    }
+
+    private boolean isDuplicatedLotteryNumbers(List<LotteryNumber> lotteryNumbers) {
+        long filteredLotteryNumberCounts = lotteryNumbers.stream()
+                .map(LotteryNumber::getLotteryNumber)
+                .distinct()
+                .count();
+        if (lotteryNumbers.size() != filteredLotteryNumberCounts) {
```

Comment:
```suggestion
        return lotteryNumbers.size() != filteredLotteryNumberCounts;
```
와 같이 좀 더 축약해볼 수 도 있을 것 같습니다 :) , 필수 사항은 아닙니다!